compose/compose-file/index.md

```diff
@@ -808,10 +812,15 @@ services:
           - spread: node.labels.zone
 ```
 
-#### replicas
+#### max_replicas_per_node
```

Comment:
I swapped the order with `replicas`, because `max_replicas_per_node` is under `placement` (so not at the same level as `replicas`